tests/sources/xml/test_dspace_mets.py

```diff
@@ -4,13 +4,15 @@
 from transmogrifier.sources.xml.dspace_mets import DspaceMets
 
 
-def create_dspace_mets_source_record_stub(xml_insert: str = "") -> BeautifulSoup:
+def create_dspace_mets_source_record_stub(
+    dmdsec_insert: str = "", filesec_insert: str = ""
```

Comment:
I think this is a nice adjustment of the stub to accomodate more sections.  This feels like a perfect way to incrementally adjust complexity of the stub _as needed_, and not before.